frontend/src/app/application-noncommercial-group/application-noncommercial-group.component.html

```diff
@@ -0,0 +1,316 @@
+<section class="usa-section">
+  <div class="usa-grid">
+    <h1>Apply for a Noncommercial Group Use Permit</h1>
+    <p>All form fields are required unless otherwise noted.</p>
+    <form class="usa-form-large" #form="ngForm" (ngSubmit)="onSubmit(form)">
+      <fieldset>
+        <legend>Applicant Information</legend>
+
+        <div *ngIf="form._submitted && !form.valid" class="usa-alert usa-alert-error" role="alert">
+          <div class="usa-alert-body">
+            <h3 class="usa-alert-heading">There are errors in your application</h3>
+            <p class="usa-alert-text">Please correct the errors before submitting.</p>
+          </div>
+        </div>
+
+        <label class="usa-input" for="name">Event name</label>
+        <input id="name" name="name" type="text" required [(ngModel)]="application.eventName" #name="ngModel" />
+        <span class="usa-input-error-message" role="alert" [hidden]="name.valid || name.untouched">Event name is required</span>
+
+        <fieldset>
```

Comment:
Good idea, and will probably be something we'll implement when we start creating the temp-outfitters form.